### DISCUSSION TITLE
MazPicker close on selection

### DISCUSSION THREAD

**Discussion by sayedwasim** [LouisMazel/maz-ui#897] [Q&A]
Hi,

I'm using the MazPicker range picker with shortcuts. I was wondering if there is a way of closing the picker on selecting a shortcut and when selecting the "from" date. I have tried looking for a solution everywhere but couldn't find one.

Hopefully looking for a solution.

Best regards,
Wasim

**Comment by LouisMazel** [comment]
Hi @sayedwasim,

I just released the [3.34.1]()

Now, the range mode supports the `auto-close` feature :

```html
<MazPicker
  v-model="rangeValues"
  label="Select periode"
  color="secondary"
  double
  auto-close
/>
```

You can test it in the documentation (range picker section)
